### PR TITLE
Improve condition so SPLIT_FASTQ skipped if only concoct selected as …

### LINKED ISSUE #590: Pipeline fails if only CONCOCT is turned on and other binners off
### Description of the bug

Fails due to the if/else statement around input to SPLIT_FASTA in binning.nf only consideres maxbin2 and/or metabat or both (CONCOCT does not produce 'unbins'). Should add logic to account for this, and allow pipeline to continue even after without SPLIT_FASTA

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by jfy133**
Closes #590 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by itrujnara — APPROVED** [review]
Looking good

File: subworkflows/local/binning.nf [hunk 1]
```diff
@@ -116,11 +118,11 @@ workflow BINNING {
     ch_split_fasta_results_transposed = SPLIT_FASTA.out.unbinned.transpose()
     ch_versions = ch_versions.mix(SPLIT_FASTA.out.versions)
 
-    GUNZIP_BINS ( ch_final_bins_for_gunzip )
+    GUNZIP_BINS ( ch_final_bins_for_gunzip.dump(tag: 'final_bins') )
```
Comment (jfy133) [inline subworkflows/local/binning.nf:121]:
```suggestion
    GUNZIP_BINS ( ch_final_bins_for_gunzip )
```

File: subworkflows/local/binning.nf [hunk 2]
```diff
@@ -116,11 +118,11 @@ workflow BINNING {
...
     ch_binning_results_gunzipped = GUNZIP_BINS.out.gunzip
         .groupTuple(by: 0)
 
-    GUNZIP_UNBINS ( ch_split_fasta_results_transposed )
+    GUNZIP_UNBINS ( ch_split_fasta_results_transposed.dump(tag: 'final_unbins') )
```
Comment (jfy133) [inline subworkflows/local/binning.nf:125]:
```suggestion
    GUNZIP_UNBINS ( ch_split_fasta_results_transposed )
```

**Comment by jfy133** [comment]
Thanks @itrujnara !

**Pull request merged**